seqr/views/apis/individual_api.py

```diff
@@ -754,7 +754,7 @@ def save_individuals_metadata_table_handler(request, project_guid, upload_file_i
 def get_individual_rna_seq_data(request, individual_guid):
     individual = Individual.objects.get(guid=individual_guid)
     check_project_permissions(individual.family.project, request.user)
-    sample = Sample.objects.get(individual=individual, is_active=True, sample_type=Sample.SAMPLE_TYPE_RNA)
+    sample = Sample.objects.get(individual=individual, is_active=True, sample_type=Sample.SAMPLE_TYPE_RNA, tissue_type=None)
```

Comment:
this is not a stable way of determining if a sample has outrider data or not. There is a chance in the future we would add other RNA data without a tissue type, or would start adding tissue type to outlier data. It would be better to skip the query for the sample altogether and just directly query the `RnaSeqOutlier` models instead - `RnaSeqOutlier.objects.filter(sample__individual=individual, sample__is_active=True, sample__sample_type=Sample.SAMPLE_TYPE_RNA)`